tests/snmp/test_snmp_queue_counters.py

```diff
@@ -58,17 +61,37 @@ def get_queue_cntrs_oid(interface):
     return queue_cntrs_oid
 
 
-def get_asic_interface(inter_facts):
+def get_dpu_npu_port_list(duthost):
```

Comment:
@aronovic this can be simplified. You can use config_facts_ports.get(k, {}).get("role", "") to get the port role.